README_PINN.md

```diff
@@ -1142,10 +1142,16 @@ cd libarchive-3.3.1
 ./configure
 make
 make install
-
+```
 if make install doesn't work (Raspbian), just copy bsdtar to /usr/bin.
+
+
+On Ubuntu 19.04+, Debian/Raspbian 10+ you can install bsdtar with
```

Comment:
According to https://packages.ubuntu.com/xenial/bsdtar it's included from Ubuntu 16.04 onwards?

Reply:
I didn't know if the version is important, xenial contains 3.1.2

`Also needed is bsdtar v3.3.1 which can be built as follows:`

Reply:
:man_shrugging: ping @procount

Reply:
I think v3.3.1 is the minimum version because it includes GNU tar compatibility and the extended attributes necessary for certain OSes.